src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {ExtensionMessage, MessagingChannel} from "../common/extensionMessaging";
+import {InterProcessMessageSender} from "../common/interProcessMessageSender";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import * as path from "path";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+try {
+    const fileInfo = process.argv[2].split(":");
+    const filename = fileInfo[0];
+    let lineNumber = "";
+    if (fileInfo.length >= 2) lineNumber = fileInfo[1];
+
+    getReactNativeWorkspaceForFile(filename).then(projectRootPath => {
```

Comment:
You should finish all promise chains calling .done() so rejected promises that are not handled by anyone, will get thrown as unhandled exceptions